### PR TITLE
Add org.freedesktop.Sdk.Extension.dotnet8

### PR DESCRIPTION
**Pull request by Dryadxon**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+Then add build options:
+
+```json
+"build-options": {
+    "arch": {
+        "arm": {
+            "env" : {
+                "RUNTIME": "linux-arm"
+            }
+        },
```
Comment (hfiguiere) [inline README.md:107]:
```suggestion
```

no need to suggest a platform that is no longer supported.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    "dest": "nuget-sources",
+    "dest-filename": "microsoft.aspnetcore.app.runtime.linux-x64.8.0.0.nupkg"
+},
+{
+    "type": "file",
+    "url": "https://api.nuget.org/v3-flatcontainer/microsoft.netcore.app.runtime.linux-arm/8.0.0-rc.1.23421.29/microsoft.netcore.app.runtime.linux-arm.8.0.0-rc.1.23421.29.nupkg",
+    "sha512": "f97e971886b99a3aea238e299c57e27096c4d1f6c666f0b029606c94ae5118a08b45ae1121561acd9c698f76153dff27c7ae845aa2226ced7ccd58ffe011ae17",
+    "dest": "nuget-sources",
+    "dest-filename": "microsoft.netcore.app.runtime.linux-arm.8.0.0.nupkg"
+},
```
Comment (hfiguiere) [inline README.md:81]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+First you need to have following lines in `sources.json`. These packages are needed to build a project for specific runtime. 
+
+```json
+{
+    "type": "file",
+    "url": "https://api.nuget.org/v3-flatcontainer/microsoft.aspnetcore.app.runtime.linux-arm/8.0.0-rc.1.23421.29/microsoft.aspnetcore.app.runtime.linux-arm.8.0.0-rc.1.23421.29.nupkg",
+    "sha512": "ca3bcb64baf61318e71506571fa047b5dda231cbeb45345cbe9e823dc8acfecedc2a9ede51a5b7e03cfa61c3acce24f54f7a44750592ec1d85be89ad34a67d5b",
+    "dest": "nuget-sources",
+    "dest-filename": "microsoft.aspnetcore.app.runtime.linux-arm.8.0.0.nupkg"
+},
```
Comment (hfiguiere) [inline README.md:60]:
```suggestion
```

File: .github/workflows/update.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+name: Check for updates
```
Comment (hfiguiere) [inline .github/workflows/update.yaml:1]: this is usually taken care by adding the proper stanza in the manifest.

File: README.md (hunk 1 above)
Comment (Dryadxon) [reply README.md:107]: Right, I forgot to remove it.

File: .github/workflows/update.yaml (hunk 4 above)
Comment (Dryadxon) [reply .github/workflows/update.yaml:1]: Not entirely sure what you're saying, this custom workflow is here to update future non-main branches, where each one is dedicated to a specific runtime version: 23.08, 24.08...
Can this task be accomplished by a manifest stanza?

Comment (Dryadxon) [reply .github/workflows/update.yaml:1]: I knew this was the only way to have multiple builds of the same package for different runtime versions.

Comment (hfiguiere) [reply .github/workflows/update.yaml:1]: nevermind.

**Comment by hfiguiere** [comment]
bot, build org.freedesktop.Sdk.Extension.dotnet8

**Pull request closed without merging**